RecoLocalMuon/GEMRecHit/plugins/GEMRecHitProducer.cc

```diff
@@ -32,43 +32,39 @@ GEMRecHitProducer::GEMRecHitProducer(const ParameterSet& config)
   // Get masked- and dead-strip information from file
   applyMasking_ = config.getParameter<bool>("applyMasking");
   if (applyMasking_) {
-    if (config.existsAs<edm::FileInPath>("maskFile")) {
```

Comment:
in the `fillDescriptions` we have only ```  desc.addOptional<edm::FileInPath>("maskFile");```
So, there is no actual parameter injection and I don't think this update/PR will work.

Reply:
looking at edmConfigDump outputs I actually see `maskFile = cms.optional.FileInPath`
So, the parameter is formally there.

@cms-sw/core-l2 
Is `if (config.existsAs<edm::FileInPath>("maskFile")) ` supposed to return `true` in this case?
If so, does the access to the value throw?